### DISCUSSION TITLE
Issue with Login provider not found

### DISCUSSION THREAD

**Discussion by MarcosRochaPrimordial** [abacritt/angularx-social-login#518] [General]
Hello,

I am getting this error on trying to login with google:
![image](https://user-images.githubusercontent.com/18124267/174504344-ba2fbd5a-ad86-4a97-89c1-764a4660ae6f.png)

At the module:
![image](https://user-images.githubusercontent.com/18124267/174504392-8e507cd4-fa85-4054-b774-c1fad9628d1b.png)

At the component:
![image](https://user-images.githubusercontent.com/18124267/174504416-1d32c94b-fcb9-4ce3-a51d-64e1de2adc20.png)

**Comment by Heatmanofurioso** [comment]
Hi @MarcosRochaPrimordial What version of the library are you using?

**Reply by MarcosRochaPrimordial** [reply]
The newest version according to NPM: 1.2.0

**Comment by hishivamku** [comment]
The same issue I am facing, it automatically shows the logged-in users in a card near the top right corner, if no user is logged in the pop-up doesn't show up.

Also, on clicking on the signin button and firing the signin method, the login window for Google doesn't show up. It shows the error shown.
 Help out!

**Comment by Jin-K** [comment]
@MarcosRochaPrimordial @Takshaksh  the signIn() function is no longer supported for Google :'(
This is handled by the new Google Identity Services js sdk now, and they want to render the login button themselves.

You can add[ `<asl-google-signin-button></asl-google-signin-button>`](https://github.com/abacritt/angularx-social-login/blob/master/src/app/demo/demo.component.html#L9) somewhere in one of your components template as "workaround", or do something similar to [this](https://github.com/abacritt/angularx-social-login/blob/master/projects/lib/src/directives/google-signin-button.directive.ts#L14-L18) replacing `el.nativeElement` with `document.getElementById('yourElemId')` for example.

It's a regression we know it but we needed a fix asap, the idea now is to work on and prepare an alternative provider for Google that gives us a way again to call that signIn() method manually and thus, have our own button to do it.

**Reply by hishivamku** [reply]
Thank you for the  workaround, I am able to use the <asl-google-signin-button>, would you mind showing me how to replace el.nativeElement, 

What I need is a custom button that I will be styling, to trigger the Google sign-in button.
